bciers/apps/registration/app/components/datagrid/models/operationGroupColumns.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  GridColumnGroupHeaderParams,
+  GridColumnGroupingModel,
+} from "@mui/x-data-grid";
+import EmptyGroupCell from "@bciers/components/datagrid/cells/EmptyGroupCell";
+import { OPERATOR_COLUMN_INDEX } from "@/app/components/datagrid/models/operationColumns";
+
+const operationGroupColumns = (
+  isInternalUser: boolean,
+  SearchCell: (params: GridColumnGroupHeaderParams) => JSX.Element,
+) => {
+  const columnGroupModel = [
+    {
+      groupId: "bcghg_id",
+      headerName: "BC GHG ID",
+      renderHeaderGroup: SearchCell,
+      children: [{ field: "bcghg_id" }],
+    },
+    {
+      groupId: "name",
+      headerName: "Operation Name",
+      renderHeaderGroup: SearchCell,
+      children: [{ field: "name" }],
+    },
+    {
+      groupId: "type",
+      headerName: "Operation Type",
+      renderHeaderGroup: SearchCell,
+      children: [{ field: "type" }],
+    },
+
+    {
+      groupId: "action",
+      headerName: "Action",
+      renderHeaderGroup: EmptyGroupCell,
+      children: [{ field: "action" }],
+    },
+  ] as GridColumnGroupingModel;
```

Comment:
Roasting my previous code here again, we should declare this type:
`const operationGroupColumns: GridColumnGroupingModel = (`